src/main/java/org/cloudfoundry/autosleep/config/Config.java

```diff
@@ -10,7 +10,10 @@
     Duration delayBeforeFirstServiceCheck = Duration.ofSeconds(10);
 
     interface Path {
-        String dashboardPrefix = "/dashboard/";
+        String apiContext = "/api";
```

Comment:
Would it make sense to use the common UPPER_CASE naming convention for these constants ?